docs/mint.json

```diff
@@ -241,6 +241,16 @@
         "api-reference/lab-testing/area-info"
       ]
     },
+    {
+      "group": "Lab Testing - At-home Phlebotomy",
+      "pages": [
+        "api-reference/lab-testing-at-home-phlebotomy/appointment-availability",
+        "api-reference/lab-testing-at-home-phlebotomy/appointment-booking",
+        "api-reference/lab-testing-at-home-phlebotomy/appointment-rescheduling",
+        "api-reference/lab-testing-at-home-phlebotomy/appointment-cancelling",
+        "api-reference/lab-testing-at-home-phlebotomy/get-appointment"
```

Comment:
appointment-cancellation-reason is not in this list

Reply:
Great catch, thanks!